css/store.css

```diff
@@ -0,0 +1,261 @@
+  font-size: 3vw;
+}
+
+h3 {
+  font-size: 2vw;
+}
+hr {
+  width: 80%;
+  border: 1px solid rgb(233, 224, 224);
+  margin: 0 auto;
+}
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+/* ======================Header styles ========================*/
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  padding: 1em;
+  border-bottom: 1px solid rgba(128, 128, 128, 0.295);
+}
+
+.nav {
+  width: 70%;
+  font-size: 1.5vw;
+  align-self: center;
+}
+
+.nav ul {
+  display: flex;
+  justify-content: right;
+  padding: 0 0.5em;
+}
+
+.nav_li {
+  list-style: none;
+  font-weight: bolder;
+  margin-right: 10px;
+  cursor: pointer;
+}
+
+.nav_link {
+  text-decoration: none;
+  color: rgb(114, 113, 113);
+}
+.nav_link:hover {
+  color: rgb(255, 115, 0);
+}
+.nav_link:active {
+  color: black;
+}
+
+.logo_div {
+  padding-left: 3%;
+}
+
+.logo_div img {
+  width: 10%;
+  max-width: 50%;
+  margin-left: 10px;
+}
+
+/* ==================================Main Styles=============================== */
+/* ------------------Store Article */
+.store {
+  width: 100%;
+}
+.store_section {
+  width: 100%;
+  display: flex;
+}
+.store_form_div {
+  width: 50%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+
+  padding: 1em 4em;
+}
+.store_form p {
+  font-size: 1.5vw;
+}
+.store_img_div {
+  width: 50%;
+  background-image: url("../img/store-image_by-andrew-neel-unsplash.jpg");
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+.store_form_div h2 {
+  color: rgb(250, 94, 32);
+}
+.input {
+  font-size: 1.2vw;
+  border: 1px solid gray;
+  padding: 0.5em;
+  margin-top: 5px;
+}
+.names_item {
+  display: flex;
+  margin-top: 5%;
+  justify-content: space-between;
+}
+.names_item label input {
+  width: 100%;
+}
+
+.address_first_item {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+.address_first_item input {
+  width: 100%;
+}
+.address_second_item {
+  display: flex;
+  justify-content: space-between;
+}
+.address_second_item label:first-child {
+  width: 100%;
+}
+.address_second_item label:first-child select {
+  width: 100%;
+}
+
+label {
+  display: block;
+  margin: 2%;
+  font-size: 1.5vw;
+}
+select {
+  border: 1px solid gray;
+  color: gray;
+}
+select,
+option {
+  margin-top: 5px;
+  padding: 0.5em;
+  font-size: 1.2vw;
+
+  background: white;
+}
+select option:first-child {
+  color: gray;
+}
+option {
+  font-size: 2vw;
+  color: black;
+}
+.color_terms_div {
+  width: 100%;
+  flex-direction: column;
+}
+.radio_part {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  margin: 5% 0;
+}
+.radio_item {
+  width: 70%;
+  display: flex;
+}
+.terms_link {
+  color: #e0633a;
+}
+input[type="radio"] {
+  width: 20px;
+  height: 20px;
+  appearance: none;
+  border: 1px solid #e08a6e;
+  border-radius: 50%;
+  padding: 3px;
+  background: #e9e1e1;
+  background-clip: content-box !important;
```

Comment:
Looks like you're using !important in a few places - usually it's a good idea to avoid this. There's more information on this here - https://uxengineer.com/css-specificity-avoid-important-css/. Can you think of a way to avoid using !important here?